src/Symfony/Component/Cache/Tests/Adapter/DoctrineDbalAdapterTest.php

```diff
@@ -107,30 +104,55 @@ public function testConfigureSchemaTableExists()
     }
 
     /**
-     * @dataProvider provideDsn
+     * @dataProvider provideDsnWithSQLite
      */
-    public function testDsn(string $dsn, string $file = null)
+    public function testDsnWithSQLite(string $dsn, string $pdoDsn, string $file = null)
     {
         try {
             $pool = new DoctrineDbalAdapter($dsn);
-            $pool->createTable();
 
             $item = $pool->getItem('key');
             $item->set('value');
             $this->assertTrue($pool->save($item));
         } finally {
+            $pdo = new \PDO($pdoDsn);
+            $pdo->exec('DROP TABLE IF EXISTS cache_items');
```

Comment:
Was there a problem with deleting the DB file just after?

Reply:
No, it just makes more sense to do the drop before deleting the file, otherwise the db is already gone when the sql is executed (if it's an SQLite file).

Reply:
There's no point in deleting the table if the entire database file is deleted subsequently.

Reply:
Sorry, completely misunderstood the question, you're right, it's not needed any more, see https://github.com/symfony/symfony/compare/b6e48698c382744f28713eaf05dc81690b786365..6a4254d90a3f8aab8728a9ac3ad08776b7df56d6

Reply:
And https://github.com/symfony/symfony/compare/6a4254d90a3f8aab8728a9ac3ad08776b7df56d6..b75880ec4d38d951fe48e1612ca77bf976077d2e